app/api/resources/mentorship_relation.py

```diff
@@ -54,7 +54,8 @@ class SendRequest(Resource):
     )
     @mentorship_relation_ns.response(
         HTTPStatus.NOT_FOUND,
-        f"{messages.MENTOR_DOES_NOT_EXIST}\n" f"{messages.MENTEE_DOES_NOT_EXIST}",
+        f"{messages.MENTOR_DOES_NOT_EXIST}\n"
+        f"{messages.MENTEE_DOES_NOT_EXIST}",
```

Comment:
```suggestion
        f"{messages.MENTOR_DOES_NOT_EXIST}\n{messages.MENTEE_DOES_NOT_EXIST}",
```

Reply:
@devkapilbansal Thank you. I made the change according to Black, which was almost identical to your suggested change. It seems that Black prefers that line as it was before my original PR :)

Reply:
Write it like this. Black will break it in two lines

Reply:
It's interesting, because that's not what happened. In my original PR, I split the line into two lines (as described in the original PR title and description). That PR failed the Black test because Black formatted the line as it is in my current PR (which subsequently passed the Black test).  Human formatting vs automated formatting I guess? haha :)